server/game/cards/02_SHD/GroguIrresistible.ts

```diff
@@ -15,6 +15,8 @@ export default class GroguIrresistible extends NonLeaderUnitCard {
             title: 'Exhaust an enemy unit',
             cost: AbilityHelper.costs.exhaustSelf(),
             targetResolver: {
+                locationFilter: WildcardLocation.AnyArena,
+                cardCondition: (card) => card.controller !== this.controller,
```

Comment:
Is the AnyArena needed? And Is this not redundant with the controller check below?

Reply:
It's needed to prevent the ability from being able to target undeployed leaders

Reply:
Oh I missed the controller comment though. Yeah that was a mistake, good catch